section4/exercises/classMethods.js

```diff
@@ -28,13 +28,33 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
-
   // ADD CODE
-};
+  changeProtein() {
```

Comment:
The `changeProtein` method should accept one argument - so that `this.protein` is not always assigned a value of `"pork"`.